hcl2template/types.build.go

```diff
@@ -130,6 +137,13 @@ func (p *Parser) decodeBuildConfig(block *hcl.Block, cfg *PackerConfig) (*BuildB
 				continue
 			}
 			build.ProvisionerBlocks = append(build.ProvisionerBlocks, p)
+		case buildErrorCleanupProvisionerLabel:
+			p, moreDiags := p.decodeProvisioner(block, cfg)
+			diags = append(diags, moreDiags...)
+			if moreDiags.HasErrors() {
+				continue
+			}
+			build.ErrorCleanupProvisionerBlock = p
```

Comment:
What happens when someone puts two of these, should we err ?

Reply:
Oh yeah, I think we should! Good catch.